### DISCUSSION TITLE
Unexpected result

### DISCUSSION THREAD

**Discussion by piso16** [svsdval/video2midi#47] [General]
Hello, first of all thank you for your work. I have a problem, I can't get the exact result from the YouTube video. this is the video --> https://www.youtube.com/watch?v=aCJxjDzyAT8 instead this is the result I get --->  https://www.swisstransfer.com/d/6b8a1c9d-787d-4d9c-9deb-f9f1f39dbefc   . I hope you can advise me on the best settings to obtain the same result in the video.

**Comment by svsdval** [comment]
Checkout the last video2midi update & use sparks
[Alan Walker - Faded - EASY Piano Tutorial by PlutaX [aCJxjDzyAT8].zip](https://github.com/svsdval/video2midi/files/13852657/Alan.Walker.-.Faded.-.EASY.Piano.Tutorial.by.PlutaX.aCJxjDzyAT8.zip)

**Comment by piso16** [comment]
The result improves with sparks but some notes that are contained within the video are missing. I also tried with another video ----> https://www.youtube.com/watch?v=GupqmKv-kWw&t=37s but the result gets worse ----> https://www.swisstransfer.com/d/a3a89acf-fd17-462f-8c8f-dd4152556f47 , I don't know if it's due to my settings (I tried with sparks and without). Very thanks
